Misc/NEWS.d/next/Core and Builtins/2020-08-31-11-37-59.bpo-41670.vmRJRx.rst

```diff
@@ -0,0 +1,2 @@
+Prevent line trace being skipped on Windows and Mac when a for loop is
+nested within a try-except block.
```

Comment:
I would mention "Python compiled without USE_COMPUTED_GOTOS" instead because there isn't anything platform-specific AFAIK

Reply:
Computed gotos are a GCC specific feature, and is always on the release builds on platforms that use GCC.
We don't use GCC for Windows or Mac.
So it is kinda platform specific.
As far as the user is concerned, it's the platform they are using, not the compiler that we use that's relevant.

Reply:
> Computed gotos are a GCC specific feature, and is always on the release builds on platforms that use GCC.

computed gotos (the capability) is actually a C language extension supported on many compilers (including gcc, clang, icc...but not the Microsoft compiler). It can perfectly be activated in Mac with clang for instance.

Reply:
Also I think we're already using computed gotos on mac according to @nedbat's comment here: https://github.com/nedbat/coveragepy/issues/959#issuecomment-600938795